homeassistant/components/media_player/philips_js.py

```diff
@@ -165,6 +167,13 @@ def mute_volume(self, mute):
         if not self._tv.on:
             self._state = STATE_OFF
 
+    def set_volume_level(self, volume):
+        """Set volume level, range 0..1."""
+        if self._volume != volume:
```

Comment:
Don't do this. It might be that our state is out of sync. It is up to the user to not call `set_volume_level` if it shouldn't.

Reply:
Done.